### ISSUE TITLE
Add keyboard support

### ISSUE THREAD

**Issue by datenhahn** [datenhahn/componentrenderer#5]
Keyboard navigation should be supported
- with ENTER set focus to the component
- with ESC leave component focus and focus the cell

**Comment by datenhahn** [comment]
keyboard support added
